src/components/Pagination/Pagination.js

```diff
@@ -102,25 +122,68 @@ export default class Pagination extends Component {
     this.props.onChange({ page, pageSize: this.state.pageSize });
   };
 
+  getItemsText = () => {
```

Comment:
Are these extra class methods necessary for #736 or are they extra on top of that issue?

Reply:
They are just extracted from inline rendering to its separate method. This was done since adding the logic for page === 0 would need one more ternary operator to existing ternary operator. I felt this would improve readability of the cod.e

Reply:
Totally understand! More-so was just curious if they are needed to close out #736, or if they are additional behavior added on.